example/lib/main.dart

```diff
@@ -95,6 +95,28 @@ class _MyAppState extends State<MyApp> {
     }
   }
   
+  void writeFileOnButtonPress() async {
+    var creds = ClientCredentials(apiKey: "", 
+                                  apiSecret: "",
+                                  clientId: "",
+                                  publicKey: "",
+                                  privateKey: "",
+                                  publicSignKey: "",
+                                  privateSigningKey: "",
+                                  host: "",
+                                  email: "",
+                                  clientName: "");
+    var client = PluginTozny(creds);
+    try {
+      RecordMeta writtenFileMeta = await writeFile(client);
+      // verify the record was created & can be read
+      Record fileRecord = await readRecord(writtenFileMeta.recordID, client);
```

Comment:
Is there a way to see that this file is actually in the file system?

Reply:
@efabens it turns out i will actually need to make a readFile method to actually download the file, so for now this is just verifying that the record actually exists. once i do that, i'll be able to check that the file is in the file system.

Reply:
👍🏻